### PR TITLE
added packages/lbvs_consent/lbvs_consent.2.1.0

### PR DESCRIPTION
**Pull request by UnixJunkie**

### CODE REVIEW

**Comment by mseri** [comment]
Fails with
```
- lbvs_consent.2.1.0:
            https://github.com/UnixJunkie/consent/archive/v2.1.0.tar.gz (Bad checksum, expected md5=e8ce325d341a0d66b686c96017c9217f)
```

File: packages/lbvs_consent/lbvs_consent.2.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,47 @@
+opam-version: "2.0"
+authors: "Francois Berenger"
+maintainer: "unixjunkie@sdf.org"
+homepage: "https://github.com/UnixJunkie/consent"
+bug-reports: "https://github.com/UnixJunkie/consent/issues"
+dev-repo: "git+https://github.com/UnixJunkie/consent.git"
+license: "GPL"
+build: [
+  [make "-C" "src"]
+  ["jbuilder" "build" "-p" name "-j" jobs]
```
Comment (kit-ty-kate) [inline packages/lbvs_consent/lbvs_consent.2.1.0/opam]: This package depends on dune but uses jbuilder. Could it use the `dune` executable instead?

jbuilder is being slowly deprecated, see: https://dune.build/blog/second-step-deprecation/

**Comment by UnixJunkie** [comment]
Ok, I pushed some more fixes

**Comment by UnixJunkie** [comment]
Some of the CI errors:
- extunix.0.1.6 doesn't build with ocaml-4.03, 4.04, 4.05 and 4.07
- camlp4 doesn't build with ocaml-4.06

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**